testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/go_back.t.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Test the ability to drill into a feature, then go back by clicking the feature menu
+ */
+StartTest(function(t) {
+
+  t.describe('Go back via the feature menu', function(t) {
+    t.it('Drill out', function(t) {
```

Comment:
Fixed. I figure we’ll use this FT to test the drilldown back button when that’s implemented, so I renamed things accordingly.